src/fee_simulator/core/path_to_transaction.py

```
@@ -469,36 +469,55 @@ def path_to_transaction_results(
-                # uses the next leader from the address pool; "timeout" entries
-                # carry no votes, "vote" entries carry a unanimous rejection.
+                # is led by the committee's current first address; "timeout"
+                # entries carry no votes, "vote" entries carry a unanimous
+                # rejection.
                 make_rotation_votes = (
                     create_vote_rotation_votes
                     if rotation_kind == "vote"
                     else create_leader_timeout_votes
                 )
+                committee = list(normal_addresses)
                 for rot_idx in range(num_timeout_rotations):
```

Ruff (0.15.20)

[warning] Loop control variable `rot_idx` not used within loop body

Rename unused `rot_idx` to `_rot_idx`

(B007)